Update Documentation

krishvoor opened the pull request:
This PR Aims at providing a fix for #6 by updating the relevant Documentation.

krishvoor committed:
Create terraform-provider-build.md

This MarkDown file provides instructions on how to build terraform providers before performing
terraform init & apply subsequently.

krishvoor committed:
Update README.md

Added Terraform Providers sub-section under Pre-requisites.

yussufsh commented on README.md:
```diff
@@ -22,6 +22,7 @@ Run this code from either Mac or Linux (Intel) system.
 - **Git**: Please refer to the following [link](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for instructions
 on installing `git` for Linux and Mac.
 - **Terraform**: You'll need to use version `0.12.20`. Please refer to the following [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing `terraform` for Linux and Mac.
+- **Terraform Providers**: Please ensure terraform providers are built and installed on KVM host. You can follow the [Build Terraform Providers](docs/terraform-provider-build.md) guide.
```
KVM host -> Terraform client machine

krishvoor replied:
Updated

yussufsh commented on docs/terraform-provider-build.md:
```diff
@@ -0,0 +1,31 @@
+# Steps to build terraform providers
+
+Follow below steps to setup terraform providers. These steps are required to be followed before running the automation.
+
```
A note to follow below sections depending on the client arch would help.

krishvoor replied:
Updated documentation to emphasize the steps required to be followed based on Terraform Client Machine Architecture.

krishvoor committed:
Update README.md

Updated KVM Host --> Terraform Client Machine

krishvoor committed:
Update terraform-provider-build.md

Updated Markdown file to emphasize steps to be followed based on Terraform Client Machine Architecture

yussufsh reviewed (approved):
LGTM

bpradipt reviewed (approved):
lgtm
Thanks @krishvoor
